ozpricechecker/pricefinderapp/models.py

```diff
@@ -1,3 +1,115 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import User
+
 
 # Create your models here.
+class Currency(models.Model):
+    name = models.CharField(max_length=3, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Store(models.Model):
+    name = models.CharField(max_length=30, unique=True)
+    prod_base_url = models.TextField(unique=True)
```

Comment:
Any Particular reason why this must be unique? I'cant think of a specific scenario where it wouldn't be right now but I would have thought since name is Unique I thought this would just be a field.
Actually, I could imagine (theoretically) that we could be splitting a real store into multiple logical stores but having the same base URL?
Thoughts?
Basically even though it'll be most likely different for each store I wouldn't think we have to enforce it this way.

Reply:
For avoid duplication and for managing store & base URL both need to be unique.